### PR TITLE
fix: update prometheus installation docs

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
## This PR

- updates prometheus installation docs, which were outdated

### CODE REVIEW

File: prometheus/0.7.2/artifacthub-pkg.yml [hunk 1]
```diff
@@ -17,42 +17,26 @@ links:
...
-  ### Install Role and Rolebinding to permit Keptn''s prometheus-service for performing
-  operations in the Prometheus installed namespace.
+  ### Install Role and RoleBinding to permit prometheus-service for performing operations in the Prometheus installed namespace:
 
   ```bash
-
-  kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-0.6.1/deploy/role.yaml
-  -n <PROMETHEUS_NS>
-
+  kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/release-<VERSION>/deploy/role.yaml -n <PROMETHEUS_NS>
```
Comment (pchila) [inline prometheus/0.7.2/artifacthub-pkg.yml:27]: Encountered an issue with this earlier today (didn't have time to file an issue yet). The role,yaml is not available at that URL. What we probably want is 
```suggestion
  kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/<VERSION>/deploy/role.yaml -n <PROMETHEUS_NS>
```

**Review by pchila — DISMISSED** [review]
fix URL of prometheus-service role manifest

File: prometheus/0.7.2/artifacthub-pkg.yml (hunk 1 above)
Comment (christian-kreuzberger-dtx) [reply prometheus/0.7.2/artifacthub-pkg.yml:27]: Thanks, that also needs to be udpated on prometheus-service itself. will do that.

**Review by oleg-nenashev — APPROVED** [review]
The comment is addressed now

**Review by TannerGilbert — APPROVED** [review]
LGTM

**Pull request merged**